fern/pages/cpp_api/cpp-api-thrust-integrations.md

```
@@ -0,0 +1,69 @@
+
+### Exec Policy
+
+Helper class usable as a Thrust CUDA execution policy
+
+```cpp
+class exec_policy : public thrust_exec_policy_t {
+```
+
+_Source: `cpp/include/rmm/exec_policy.hpp:40`_
+
+### Exec Policy
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)